src/components/Filters/types/String.js

```diff
@@ -19,30 +19,35 @@ export const StringFilter = [
     key: "contains",
     label: "contains",
     valueType: "single",
+    hasMilvusSupport: false,
     input: (props) => <BaseInput {...props} />,
   },
   {
     key: "not_contains",
     label: "not contains",
     valueType: "single",
+    hasMilvusSupport: false,
     input: (props) => <BaseInput {...props} />,
   },
   {
     key: "regex",
     label: "regex",
     valueType: "single",
+    hasMilvusSupport: false,
     input: (props) => <BaseInput {...props} />,
   },
   {
     key: "equal",
     label: "equal",
     valueType: "single",
+    hasMilvusSupport: true,
     input: (props) => <BaseInput {...props} />,
   },
   {
     key: "not_equal",
     label: "not equal",
     valueType: "single",
+    hasMilvusSupport: true,
```

Comment:
`hasMilvusSupport`

I wouldn't add vendor specific things to DataManager, as this is then contextually tied forever like this, and this isn't available to broad audience. Filtering the operations should be done in a way that is as agnostic as possible, without putting notions of vendors into the main code.

Reply:
or just `isSupported` but that is very vague so i dont like this one as much